http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1613,6 +1613,281 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
+  "/proxy/{googleProject}/{runtimeName}/yarn":
+    get:
+      summary: YARN Resource Manager
+      description: >
+        Only available for Dataproc runtimes with Component Gateway enabled.
+        See https://cloud.google.com/dataproc/docs/concepts/accessing/dataproc-gateways
+      operationId: yarn
+      tags:
+        - proxy
+      parameters:
+        - in: path
+          name: googleProject
+          description: googleProject
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: runtimeName
+          description: runtimeName
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Proxy connection successful
+        "403":
+          description: Proxy connection unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorReport"
+        "404":
+          description: Runtime not found
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorReport"
+        "422":
+          description: Runtime is stopped
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorReport"
+        "423":
+          description: Runtime not ready
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorReport"
+        "500":
+          description: Internal Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorReport"
+  "/proxy/{googleProject}/{runtimeName}/jobhistory":
```

Comment:
I wonder if we should hyphenate URI components for easier reading as in `job-history`. That's what I mostly commonly see in API naming conventions.

Reply:
Hm, I stole the URL paths directly from Dataproc Component Gateway: https://cloud.google.com/dataproc/docs/concepts/accessing/dataproc-gateways